chart/values.schema.json

```diff
@@ -1591,6 +1591,27 @@
                             "type": "boolean",
                             "default": true
                         },
+                        "persistentVolumeClaimRetentionPolicy": {
+                            "description": "How PVCs are deleted during the lifecycle of a StatefulSet",
+                            "type": [
+                                "object",
+                                "null"
+                            ],
+                            "default": null,
+                            "additionalProperties": false,
+                            "properties": {
+                                "whenDeleted": {
+                                    "description": "Whether to retain the PVC when the StatefulSet is deleted.",
+                                    "type": "string",
+                                    "default": "Retain"
+                                },
+                                "whenScaled": {
+                                    "description": "Whether to retain the PVC when the StatefulSet is scaled.",
+                                    "type": "string",
+                                    "default": "Retain"
+                                }
+                            }
+                        },
```

Comment:
You can create a new definition in `definitions` to reuse instead of duplicating it twice.

Reply:
move to `definitions`